apps/nextjs/src/stores/moderationStore/actionFunctions/handleToxicModeration.tsx

```diff
@@ -0,0 +1,10 @@
+import type { PersistedModerationBase } from "@oakai/core/src/utils/ailaModeration/moderationSchema";
+import { useChatStore } from "src/stores/chatStore";
+
+export const handleToxicModeration = (
```

Comment:
```suggestion
export const handleFlagToxicModeration = (
```

Maybe?

Reply:
Or maybe the opposite. Maybe remove the `flag` from other places as it's not very clear what flagging is